wordpress-mobile/WordPress-iOS
# Add `OpenedWebPreview` event for web previews

Associated [WordPressShared PR](https://github.com/wordpress-mobile/WordPress-iOS-Shared/pull/241)

Adds a new analytics event for when web previews are opened. The event includes `post_type` and `blog_type` properties that mimic those used by the editor.

To test:

- Visit a blog post or site preview
- Verify that the `web_preview_opened` event is logged

PR submission checklist:

- [x] ~I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.~ Feature not released
